packages/server/src/notification/notification.service.ts

```diff
@@ -50,7 +50,7 @@ export class NotificationService {
   }
 
   // Notify user on all platforms
-  async notifyUser(userId: number) {
+  async notifyUser(userId: number, message: string): Promise<void> {
```

Comment:
Same thing here? Again, not sure if making the return type void will break the async